### PR TITLE
refactor(HyperlinkButton): lightweight styling for hyperlinkbutton

### PR DESCRIPTION
**Pull request by rajamatt**
﻿GitHub Issue: #1018 #1067

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## Description

- Moving two resources to ResourceDictionary in HyperlinkButton XAML.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [ ] Tested iOS
- [ ] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml [hunk 1]
```diff
@@ -49,15 +52,15 @@
...
             <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPressed" ResourceKey="LowOpacity" />
-            <StaticResource x:Key="HyperlinkButtonBackgroundOpacityDisabled" ResourceKey="NormalOpacity" />
-        </ResourceDictionary>
-    </ResourceDictionary.ThemeDictionaries>
+            <StaticResource x:Key="HyperlinkButtonBackgroundOpacityDisabled" ResourceKey="HyperlinkButtonNormalOpacity" />
 
-    <Thickness x:Key="HyperlinkButtonPadding">0</Thickness>
-    <x:Double x:Key="NormalOpacity">1</x:Double>
+			<Thickness x:Key="HyperlinkButtonPadding">0</Thickness>
+			<x:Double x:Key="HyperlinkButtonNormalOpacity">1</x:Double>
```
Comment (Xiaoy312) [inline src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml:61]: we could just rename this to HyperlinkButtonBackgroundOpacity, and move it up
same for disabled

Comment (rajamatt) [reply src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml:61]: like this? 

```      
<ResourceDictionary> 
        ...
        ...

        <x:Double x:Key="HyperlinkButtonBackgroundOpacity">1</x:Double>
        <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPointerOver" ResourceKey="MediumOpacity" />
        <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPressed" ResourceKey="LowOpacity" />
	<x:Double x:Key="HyperlinkButtonBackgroundOpacityDisabled">1</x:Double>

	<Thickness x:Key="HyperlinkButtonPadding">0</Thickness>
</ResourceDictionary>

Comment (kazo0) [reply src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml:61]: @Xiaoy312 is this what you meant? ^

**Review by kazo0 — APPROVED** [review]

**Comment by kazo0** [comment]
@Mergifyio backport legacy/2x

**Pull request merged**